cordformation/src/main/kotlin/net/corda/plugins/NetworkParameterOverrides.kt

```diff
@@ -1,29 +1,60 @@
 package net.corda.plugins
 
-import com.typesafe.config.Config
-import com.typesafe.config.ConfigFactory
-import com.typesafe.config.ConfigObject
-import com.typesafe.config.ConfigValueFactory
+import com.typesafe.config.*
 import org.gradle.api.Action
 import org.gradle.api.NamedDomainObjectContainer
 import org.gradle.api.Project
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Nested
+import org.gradle.api.tasks.Optional
+import java.time.Duration
 import javax.inject.Inject
 
 open class NetworkParameterOverrides @Inject constructor(project: Project) {
 
+    @get:Optional
+    @get:Input
+    var minimumPlatformVersion: Property<Int> = project.objects.property(Int::class.java)
```

Comment:
These should all be `val` instead of `var`. A "lazy" property is a container object for a value, and the _container_ will not change.